react/finger-exercises/redux/02 - react-redux/src/app/screens/App/index.js

```diff
@@ -54,23 +44,30 @@ class App extends Component {
 
   render() {
     return (
-      <Fragment>
-        <Navbar />
-        <div className={styles.container}>
-          <Search onSearch={this.onSearch} />
-          {this.state.books.length ? (
-            this.state.books.map(this.renderBooks)
-          ) : (
-            <div className={styles.noData}>
-              <h2 className={styles.title}>No Data</h2>
-            </div>
-          )}
-        </div>
-        {this.state.bookSelected.length ? (
-          <ShoppingCart data={this.state.bookSelected} addItem={this.addItem} removeItem={this.removeItem} />
-        ) : null}
-        <Footer />
-      </Fragment>
+      <Provider store={store}>
+        <Fragment>
+          <Navbar />
+          <Book key={this.state.books[0].id} data={this.state.books[0]} configButton={undefined} />
```

Comment:
no need to pass `undefined`, you may avoid the prop altogether. Also, this should crash when clicking, as undefined doesn't have an `isDanger`.